packages/lesswrong/components/users/UsersSingle.jsx

```diff
@@ -1,10 +1,18 @@
-import { Components, registerComponent } from 'meteor/vulcan:core';
+import { Components, registerComponent, Utils } from 'meteor/vulcan:core';
 import React from 'react';
+import { withRouter } from 'react-router';
 
-const UsersSingle = (props, context) => {
-  return <Components.UsersProfile userId={props.params._id} slug={props.params.slug} />
+const UsersSingle = ({params, router}) => {
```

Comment:
doesn't look like this actually addresses the "display name doesn't match" thing, which is maybe fine, still looks like a local improvement. But wanted to at least touch base on that and whether there are good options for it.

Reply:
Indeed it doesn't. This addresses the issues with capitalization, special characters, and /u/ vs /user/ vs /users/, but not display-name vs slug mismatches. Addressing slug mismatches is feasible, but is a bit more involved; we want to switch from users having a slug, to users having a set of slugs, one of which is canonical.